components/layout.tsx

```diff
@@ -81,7 +81,7 @@ const Layout: React.FC<Props> = ({
         )}
         {image && <meta property="og:image" content={image} />}
         {image && (
-          <meta property="twitter:card" content="summary_large_image" />
```

Comment:
Here's an example of metadata that was confirmed working (from the twitter example sent by a user) -- we're still missing a few things:

```html
<head>
    <meta charset="utf-8">
    <title>Mux embed test</title>
    <meta name="twitter:card" content="player">
    <meta name="twitter:title" content="Video uploaded to stream.new">
    <meta name="twitter:site" content="@yourfightsite">
    <meta name="twitter:player" content="https://stream.new/v/d9tCY7QMM6J8G00kBi1AS5dlpSPljk01PG48J2800iaWZY/embed">
    <meta name="twitter:player:width" content="480">
    <meta name="twitter:player:height" content="270">
    <meta name="twitter:image" content="https://image.mux.com/d9tCY7QMM6J8G00kBi1AS5dlpSPljk01PG48J2800iaWZY/thumbnail.jpeg?width=480">
  </head>
  ```